packages/text-area/src/vaadin-text-area-mixin.js

```diff
@@ -51,6 +60,10 @@ export const TextAreaMixin = (superClass) =>
       return [...super.constraints, 'maxlength', 'minlength', 'pattern'];
     }
 
+    static get observers() {
+      return ['__updateMinHeight(minRows, inputElement, _inputField)'];
```

Comment:
nit: we probably don't need to pass `_inputField` since it's not actually used in the observer, it should be enough to pass `inputElement` - both are defined in the `ready()` block anyway.

Reply:
Removed the argument for now. Input container will be used for `maxRows`, but as you mentioned both should be available at the same time.